Generals/Code/GameEngine/Include/Common/Recorder.h

```diff
@@ -119,6 +119,7 @@ class RecorderClass : public SubsystemInterface {
 	void initControls();															///< Show or Hide the Replay controls
 
 	AsciiString getReplayDir();												///< Returns the directory that holds the replay files.
+	AsciiString getReplayArchiveDir();								///< Returns the directory that holds the archived replay files.
 	static AsciiString getReplayExtention();									///< Returns the file extention for replay files.
```

Comment:
Maybe you can change this to return `const char*`, to avoid useless `AsciiString` allocations in many places.